sponsor.html

```diff
@@ -0,0 +1,404 @@
+          Our available sponsorship tiers are listed on the following page.
+          Additionally, we are happy to state that Technica has <b>501(c)(3)</b> status.
+          Please let us know if you are interested in in becoming a part of
+          Technica, or if you have any questions or concerns about the information
+          below. We look forward to hearing from you!
+        </p>
+        <p>
+          Thank you,
+          <br>
+          The Technica Team
+        </p>
+      </div>
+
+      <!-- contact -->
+      <div class="contact">
+        <div class="heading">
+          <h2> Let's Get In Touch </h2>
+        </div>
+        <p>Contact us at sponsorship@gotechnica.org or send an inquiry below!</p>
+        <div class="inquiry">
+          <form>
+            <div class="info-input">
+              <div class="form-row">
+                <div class="form-group col-md-8">
+                  <label class="sr-only" for="name-input">Full Name</label>
+                  <input type="text" class="form-control" id="name-input" placeholder="Full Name">
+                </div>
+                <div class="form-group col-md-4">
+                  <label class="sr-only" for="company-input">Company Name</label>
+                  <input type="text" class="form-control" id="company-input" placeholder="Company Name">
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="sr-only" for="sponsor-email-input">Email Address</label>
+                <input type="email" class="form-control" id="sponsor-email-input" placeholder="Email Address">
+              </div>
+            </div>
+            <div class="form-group">
+              <label class="sr-only" for="inquiry-input">Inquiry</label>
+              <input type="text" class="form-control" id="inquiry-input" placeholder="Inquiry">
+            </div>
+            <button type="submit" class="btn">Send</button>
+          </form>
+        </div>
+      </div>
+
+      <!-- stats -->
+      <div class="stats">
+        <div class="heading">
+          <h2> Statistics </h2>
+        </div>
+        <div class="overall-stats">
+          <div class="row">
+            <div class="col-md-5">
+              <div class="row">
+                <div class="col-md-6">850+<br>Attendees</div>
+                <div class="col-md-6">150+<br>Schools</div>
+              </div>
+              <div class="row">
+                <div class="col-md-6">144<br>Projects submitted</div>
+                <div class="col-md-6">37<br>Sponsors</div>
+              </div>
+            </div>
+            <div class="col-md-7">
+              img placeholder
+            </div>
+          </div>
+        </div>
+        <div class="demographics">
+          <div class="text-center">
+            <h3>Demographics</h3>
+          </div>
+          <div class="row">
+            <div class="col-md-4">
+              education lvl graph placeholder
```

Comment:
Not going to worry about the graph part right now since you're still working on it, it seems